hoot-core/src/main/cpp/hoot/core/cmd/InfoCmd.cpp

```diff
@@ -119,11 +119,11 @@ class InfoCmd : public BaseCommand
 
       if (args.size() == 0)
       {
-        ConfigOptionsDisplayer::displayAllOptionNames(getDetails);
+        std::cout << ConfigOptionsDisplayer::getAllOptionNames(getDetails).toStdString();
```

Comment:
Changed these functions to return strings that are then output in the command class instead of spread throughout the code.